homeassistant/components/blink/alarm_control_panel.py

```diff
@@ -32,25 +33,29 @@ async def async_setup_entry(
     hass: HomeAssistant, config: ConfigEntry, async_add_entities: AddEntitiesCallback
 ) -> None:
     """Set up the Blink Alarm Control Panels."""
-    data = hass.data[DOMAIN][config.entry_id]
+    coordinator: BlinkUpdateCoordinator = hass.data[DOMAIN][config.entry_id]
 
     sync_modules = []
-    for sync_name, sync_module in data.sync.items():
-        sync_modules.append(BlinkSyncModuleHA(data, sync_name, sync_module))
-    async_add_entities(sync_modules, update_before_add=True)
+    for sync_name, sync_module in coordinator.api.sync.items():
+        sync_modules.append(BlinkSyncModuleHA(coordinator, sync_name, sync_module))
+    async_add_entities(sync_modules)
 
 
-class BlinkSyncModuleHA(AlarmControlPanelEntity):
+class BlinkSyncModuleHA(CoordinatorEntity, AlarmControlPanelEntity):
```

Comment:
```suggestion
class BlinkSyncModuleHA(CoordinatorEntity[BlinkUpdateCoordinator], AlarmControlPanelEntity):
```

Reply:
Side note, I think trying to extract a common `BlinkEntity` would maybe help to default store stuff like coordinator and do device info. But please do that in a separate PR. (I think this might be convenient to do first, but that's up to you)